binr/radare2/radare2.c

```diff
@@ -1081,8 +1081,13 @@ int main(int argc, char **argv, char **envp) {
 				f = r_acp_to_utf8 (f);
 #	endif // __WINDOWS__
 				if (f) {
-					char *escaped_path = r_str_arg_escape (f);
-					pfile = r_str_append (pfile, escaped_path);
+					char *escaped_path = NULL;
+#		if !__WINDOWS__
```

Comment:
what about duplicating the whole body like this, looks nicer to read imho:

#if __WINDOWS__
char *escaped_path = NULL;
#else 
char *escaped_path = r_str_arg_escape (f);
#endif

Reply:
well in fact this can be 2 statements with pfile = r_str_append (pfile, r_str_arg_escape (f))